lib/recording/RecordingManager.js

```diff
@@ -54,7 +54,7 @@ class RecordingManager {
     getRecordingEntrySession(entryObj, mp4FilesQueue = null) {
         let entryId = entryObj.entryId;
         if (mp4FilesQueue != null && mp4FilesQueue.length > 0) {
-            logger.debug(`[${entryObj.entryId}-${entryObj.recordedEntryId}] getRecordingEntrySession called with mp4 files queue size [${mp4FilesQueue.toString()}]`);
+            logger.debug(`[${entryObj.entryId}-${entryObj.recordedEntryId}] getRecordingEntrySession called with mp4 files queue size [${mp4FilesQueue.length}][${JSON.stringify(mp4FilesQueue)}]`);
```

Comment:
this is dangerous  since the mp4FilesQueue could be large and this will kill the logger

Reply:
I'll leave the size and drop the content.

Reply:
printing names of chunks alone. this is less dangerous